cni/network/invoker_cns_test.go

```diff
@@ -213,31 +212,118 @@ func TestCNSIPAMInvoker_Add_Overlay(t *testing.T) {
 				hostSubnetPrefix: getCIDRNotationForAddress("10.0.0.1/24"),
 				options:          map[string]interface{}{},
 			},
-			wantIpv4Result: &cniTypesCurr.Result{
+			wantDefaultResult: &cniTypesCurr.Result{
 				IPs: []*cniTypesCurr.IPConfig{
 					{
 						Address: *getCIDRNotationForAddress("10.0.1.10/24"),
 						Gateway: net.ParseIP("10.0.0.1"),
 					},
+					{
+						Address: *getCIDRNotationForAddress("fd11:1234::1/112"),
+						Gateway: net.ParseIP("fe80::1234:5678:9abc"),
+					},
 				},
 				Routes: []*cniTypes.Route{
 					{
 						Dst: network.Ipv4DefaultRouteDstPrefix,
 						GW:  net.ParseIP("10.0.0.1"),
 					},
+					{
+						Dst: network.Ipv6DefaultRouteDstPrefix,
+						GW:  net.ParseIP("fe80::1234:5678:9abc"),
+					},
 				},
 			},
-			wantIpv6Result: &cniTypesCurr.Result{
+			wantErr: false,
+		},
+		{
+			name: "Test happy CNI add with multitenant result",
```

Comment:
is test added for covering delegated nic scenario?

Reply:
yup